src/__e2e__/packageManager.test.ts

```diff
@@ -23,8 +23,8 @@ describe('packageManager', () => {
       await registry.reset();
     });
 
-    it('can publish', () => {
-      const publishResult = packagePublish(testPackageInfo, registry.getUrl(), '', '');
+    it('can publish', async () => {
+      const publishResult = await packagePublish(testPackageInfo, registry.getUrl(), '', '');
```

Comment:
Most of the test changes are just because I made `packagePublish` return a promise, so now it needs to be awaited in the tests that use it